src/Kentico.Xperience.TagManager/Admin/UIPages/CodeSnippetListingPage.cs

```diff
@@ -49,6 +50,8 @@ internal class CodeSnippetListingPage : ListingPage
 
     public override async Task ConfigurePage()
     {
+        await base.ConfigurePage();
```

Comment:
The base page should always be awaited to prevent race conditions in the case that the base page is doing async work.